docs/examples/bower.yml

```diff
@@ -0,0 +1,5 @@
+excludes:
+  scopes:
+  - name: "devDependencies"
+    reason: "BUILD_TOOL_OF"
```

Comment:
Should be "TEST_CASE_OF" as these are usually test dependencies, not dependencies used to build the product. Adapt the description accordingly.
Same applies to the other examples.

Reply:
Bower is based on Node.js which simply does not have a test scope only a development scope. The way how I envisioned this working is that people set BUILD_TOOL_OF for all devDependencies and then use an exclude on package IDs to declare specific packages as TEST_CASE_OF. Or alternatively we allow multiple excludes scopes on the same name

Reply:
I have applied the following rules:
(1) scope name contains "test" or "Test" --> reason: "TEST_TOOL_OF"
(2) scope name equals "devDependencies" -> reason "BUILD_TOOL_OF"
(3) scope name equals "bench" -> reason "TEST_TOOL_OF"